L1Trigger/L1TTrackMatch/interface/L1TrackJetProducer.h

```diff
@@ -1,21 +1,16 @@
-#ifndef L1Trigger_L1TTrackMatch_L1TrackJetProducer_HH
-#define L1Trigger_L1TTrackMatch_L1TrackJetProducer_HH
+#pragma once
 #include <cmath>
 #include <cstdlib>
-#include <fstream>
-#include <iostream>
-#include <string>
-#include <vector>
```

Comment:
@gkaratha the compiler is complaining because you removed `#include <vector>` here and so any of the `std::vector` below is defined.

Reply:
```c++

src/L1Trigger/L1TTrackMatch/interface/L1TrackJetProducer.h:1:9: warning: #pragma once in main file
    1 | #pragma once
      |         ^~~~
src/L1Trigger/L1TTrackMatch/interface/L1TrackJetProducer.h:16:8: error: 'vector' in namespace 'std' does not name a template type
   16 |   std::vector<unsigned int> trackidx;
      |        ^~~~~~
src/L1Trigger/L1TTrackMatch/interface/L1TrackJetProducer.h:4:1: note: 'std::vector' is defined in header '<vector>'; did you forget to '#include <vector>'?
    3 | #include <cstdlib>
  +++ |+#include <vector>
```

Reply:
Ok should I added back?

Is there away from git to take this PR? (given that is Cecile's)

Reply:
Yes it should be added back but I think only @cecilecaillol may push the modification. I've added a suggestion above.